packages/dht/src/connection/websocket/AutoCertifierClientFacade.ts

```diff
@@ -77,13 +77,18 @@ export class AutoCertifierClientFacade {
             this.updateCertificate(subdomain.certificate, subdomain.privateKey)
             logger.trace(`Updated certificate`)
         })
-        await Promise.all([
-            waitForEvent3(this.autoCertifierClient as any, 'updatedCertificate', START_TIMEOUT),
-            this.autoCertifierClient.start()
-        ])
+
+        // waiting for 'updatedCertificate' event is not needed here because
+        // all paths of this.autoCertifierClient.start() will block until 'updatedCertificate' 
+        // event is emitted or an exception is thrown
+
+        await withTimeout(this.autoCertifierClient.start(), START_TIMEOUT,
```

Comment:
Maybe we should separate the starting of the client and obtaining the initial certificate. It doesn't take long time to start the client, but in practice we wait for the certificate?

Also could rename `AutoCertifierClientFacade` to `AutoCertifierFacade` as we don't seem to use any of the client's functionality methods (we just call the `start` and `stop` methods).

-> maybe in a separate PR